admin_guide/managing_projects.adoc

```diff
@@ -376,3 +376,48 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-limits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
+
+If you are an {product-title} administrator and want to enforce different limits
+on the number of projects that your users can create, apply the appropriate
+label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+
+.Example Project Limit for Users
+
+----
+openshift.io/project-limit: '0'
+openshift.io/project-limit: '1'
+openshift.io/project-limit: '3'
+openshift.io/project-limit: '5'
+openshift.io/project-limit: '10'
+openshift.io/project-limit: '20'
+openshift.io/project-limit: '50'
+openshift.io/project-limit: '100'
+openshift.io/project-limit: 'unlimited'
+----
+
+[[managing-limits-and-quota-on-project-resources]]
+== Managing Limits and Quota on Project Resources
+
+A default project template is available for any projects created by users on the
+cluster. The template helps prevent inadvertent consumption all of cluster and
+node resources. The project template is created and configured in the
+`dedicated-admin` namespace.
+
+If you modify the default project template and want your changes to remain after
+you upgrade, must update the `openshift.io/overwrite-protect` annotation value
+to `true`.
+
+----
+openshift.io/overwrite-protect:true
+----
+
+The default value is `false`, so by default your changes are not protected
+against overwrite.
```

Comment:
We need to also highlight that the annotation value need not be changed unless the customer makes changes to the template. The above line seems to give the impression that there is no downside to setting the annotation to "true". We need to highlight that setting it to "true" will result in the upgrader skipping the template and not updating it. It is then the customer's responsibility to see if the newer versions being shipped with the platform provide anything of interest that they would need to incorporate.